### ISSUE TITLE
Uncaught TypeError: Cannot read property 'firstChild' of undefined

### ISSUE THREAD

**Issue by miacrd** [atom/atom#15335]
[Enter steps to reproduce:]

Opened sidebar of conflicts in files after doing a pull on a branch... Some popups started appearing that stated the issue was already reported, so I closed the sidebar. The popups kept showing up, and then some showed up that prompted me to create an issue, so I assumed it was a different one.

Only showed up when I was on the tab of the conflicted file! I changed to tab of settings and the popups stopped, and started again when I switched back to my conflicted file. As soon as I resolved the conflict issues, the popups stopped. I still have another conflicted file to correct, will update this to see if behaviour is the same.

So it seems only after opening the git tab (where it shows on the bottom right corner of the screen how many files have conflicts) the errors occur, since I switched to my second conflicted file and was able to scroll (although unsure if the markup of "ours vs theirs" changes had updated already, since I didn't see anything). Then again the same, even after closing the tab, whenever I scroll more errors are thrown.

**Atom**: 1.19.1 ia32
**Electron**: 1.6.9
**OS**: Microsoft Windows 10 Home
**Thrown From**: Atom Core

### Stack Trace

Uncaught TypeError: Cannot read property 'firstChild' of undefined

```
At C:\Users\Marta\AppData\Local\atom\app-1.19.1\resources\app\src\text-editor-component.js:2171

TypeError: Cannot read property 'firstChild' of undefined
    at TextEditorComponent.measureLongestLineWidth (~/AppData/Local/atom/app-1.19.1/resources/app/src/text-editor-component.js:2171:112)
    at TextEditorComponent.measureContentDuringUpdateSync (~/AppData/Local/atom/app-1.19.1/resources/app/src/text-editor-component.js:383:16)
    at scheduler.readDocument (~/AppData/Local/atom/app-1.19.1/resources/app/src/text-editor-component.js:270:20)
    at ViewRegistry.module.exports.ViewRegistry.performDocumentUpdate (~/AppData/Local/atom/app-1.19.1/resources/app/src/view-registry.js:191:15)
    at ~/AppData/Local/atom/app-1.19.1/resources/app/src/view-registry.js:3:65
```

### Commands

```
```

### Non-Core Packages

```
anodyne-dark-syntax 0.2.1 
atom-monokai-php 1.0.0 
glacier-dark-ui 0.5.0 
glacier-syntax 0.0.4 
minimap 4.29.6 
monokai 0.24.0 
spectrum-dark-syntax 0.2.0 
```

**Comment by miacrd** [comment]
Also this issue popped up: https://github.com/atom/atom/issues/11745
It's marked as closed, but on the program it shows as "view issue" and I assumed it was open

**Comment by qiugits** [comment]
This issue occurred to me when I open my project after I set up a default window size and tree width by editing Styles... and Init Script... . I assume this happened because it conflicts with the size Atom auto saved when the last time I opened my project.
I hope this information helps. 
Regards,

**Comment by littleread** [comment]
+1

**Comment by nathansobo** [comment]
Investigating.

**Comment by valentinoli** [comment]
I'm also getting this error.

**Comment by WonJunior** [comment]
It happens to me too: the error message appears when **I zoom in or out**

**Comment by nathansobo** [comment]
We should have a fix out shortly with 1.19.2. Sorry for the trouble everyone.
